<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageCapturesTab</class>
 <widget class="QWidget" name="ManageCapturesTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="capturedHandshakesList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>151</width>
     <height>281</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>true</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>true</bool>
   </property>
   <property name="selectionRectVisible">
    <bool>false</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="captureInformationTextEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>431</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
